src/zeit/content/cp/centerpage.py

```diff
@@ -200,10 +200,7 @@ def _fill_area_cache(self):
     _area_teasered_content = gocept.cache.property.TransactionBoundCache(
         '_v_area_teasered_content', dict)
 
-    def is_teaser_present_above(self, current_area, content):
-        return content in self._teasered_content_above(current_area)
-
-    def _teasered_content_above(self, current_area):
+    def teasered_content_above(self, current_area):
```

Comment:
Doch, die Funktion. Und wenn man lediglich `Region.values` oder `Area.values` mockt, dann ist ja nur Zeug außerhalb der CP gemockt. Finde ich in Ordnung. Alternativ geht dann nur schwergewichtiger Test mit großer CP die echte Regionen und Areas enthält.